src/frontend/components/LTINav/index.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import { DASHBOARD_ROUTE } from '../Dashboard/route';
+import { PLAYER_ROUTE } from '../routes';
+import { appData } from '../../data/appData';
+import { Box, Button, Nav, Tab, Tabs, Text } from 'grommet';
+import { defineMessages, FormattedMessage } from 'react-intl';
+import styled from 'styled-components';
+import { withRouter } from 'react-router';
+
+const messages = defineMessages({
+  linkDashboard: {
+    defaultMessage: 'Dashboard',
+    description: `Title for the dashboard, where the user can see the status of the video/audio/timed text upload
+      & processing, and will be able to manage them.`,
+    id: 'components.LTINav.linkDashboard',
+  },
+  linkPreview: {
+    defaultMessage: 'Preview',
+    description: `Title for the Instructor View. Describes the area appearing right above, which is a preview
+      of what the student will see there.`,
+    id: 'components.LTINav.linkPreview',
+  },
+});
+
+const NavItem = styled(NavLink)`
+  cursor: pointer;
+  padding: 1rem;
+  color: #444444;
+  text-decoration: none;
+  border-bottom: solid 2px #ededed;
+
+  :hover {
+    border-bottom-color: #007bff;
+  }
+
+  &.active {
+    border-bottom-color: #000000;
+  }
+`;
+
+const links = [
+  {
+    label: messages.linkDashboard,
+    route: DASHBOARD_ROUTE,
+  },
+  {
+    label: messages.linkPreview,
+    route: PLAYER_ROUTE,
```

Comment:
The preview can not be display if the video is not ready to play. Because there is nothing to display in that case ;-)